meta/what_is_artsy.md

```diff
@@ -33,7 +33,8 @@ We believe in empowering each of our team members to feel like the CEO of their
 
 We strive to be a platform that is worthy of presenting the world’s most beautiful and culturally significant objects to all of humanity. By choosing to work at Artsy, we accept responsibility for this endeavor, and our work demonstrates a level of quality and intention that is worthy of the artists and people we serve.
 
-> **Example**: We embed product designers into our teams to work directly with engineers; collaboration between product vision and product development helps us create products that are of a quality worthy of art.
+> **Example**: We have opted to not release products that are ready to ship because of they would not be up to par based on 
+  externalities we can't control.
```

Comment:
This example could be more specific. I really like the example in the PR description of the Apple Watch app, could we add that?